src/isomorphic.ts

```diff
@@ -0,0 +1,4 @@
+/*
+ * IMPORTANT: You must only expose exports from isomorphic modules.
+ */
+export * as marshal from "./lib/marshal/index.isomorphic";
```

Comment:
This change is based on the PR feedback/discussion I had separately here: https://github.com/knocklabs/control/pull/2659#discussion_r1424607071

Instead of reaching deeply into inner modules, now we are exposing `marshal` as a named import so we can do imports via "@knocklabs/cli/dist/isomorphic" instead like [here](https://github.com/knocklabs/control/pull/2704/files#diff-ae026a672981573bca4880032da78d5f8e58152ffce73284568bf74beb3064e5L14).